charts/ocis/ci/values.yaml

```diff
@@ -27,6 +27,14 @@ backup:
     backup.test/include: "true"
     backup.test/pvc: "test"
 
+route:
+  enabled: true
```

Comment:
I'd be fine for omitting the route for liniting, since this is not a default kubernetes ressource type. Then you don't need to mess with the OpenShift json schemas.